builds/misc/images.yaml

```diff
@@ -180,6 +180,13 @@ jobs:
           imageName: azureiotedge-c2dmessage-tester
           project: CloudToDeviceMessageTester
 
+     # Azure Monitor
+      - template: templates/image-linux.yaml
+        parameters:
+          name: Metrics Collector
+          imageName: azureiotedge-metrics-collector
```

Comment:
On line 159 there is another image with the same tag being published. We depend on this for test metrics.

Reply:
@pmzara I think fixing this is necessary for merge. Lmk if you need any help. Bear and I have a ton of context here.

Reply:
So if I understand correctly, the Build Images pipeline will overwrite the old metrics collector (test) module with the new one (Azure Monitor). In that case, this PR _is_ the right place to address this...sorry for the back-and-forth. @and-rewsmith and @pmzara, can you decide on a docker image name for the test module, then update `builds/misc/images.yaml` _in this PR_ with the new name?

Reply:
I think we should go with azureiotedge-test-metrics-collector